ResearchKit/Common/ORKContinueButton.m

```diff
@@ -70,21 +79,21 @@ - (void)updateConstraints {
                                                          relatedBy:NSLayoutRelationEqual
                                                             toItem:nil
                                                          attribute:NSLayoutAttributeNotAnAttribute
-                                                        multiplier:1
-                                                          constant:44];
+                                                        multiplier:1.0
+                                                          constant:0.0];
         _heightConstraint.active = YES;
     }
     if (! _widthConstraint) {
-        UIWindow *window = [self window];
-        ORKScreenType screenType = ORKGetScreenTypeForWindow(window);
         _widthConstraint = [NSLayoutConstraint constraintWithItem:self
                                                         attribute:NSLayoutAttributeWidth
                                                         relatedBy:NSLayoutRelationGreaterThanOrEqual
                                                            toItem:nil
                                                         attribute:NSLayoutAttributeNotAnAttribute
-                                                       multiplier:1
-                                                         constant:ORKGetMetricForScreenType(ORKScreenMetricContinueButtonWidth, screenType)];
+                                                       multiplier:1.0
+                                                         constant:0.0];
```

Comment:
Can you add comment here to indicate the constant will be adjusted in `updateConstraintConstants` ?